### PR TITLE
Beginning of the ts files

### PR DESCRIPTION
**Pull request by moreiravictor**
Just creating models (should get better later), starting the development of the index.ts and taking most of db configs out of the index, as well as using some imports on the ts pattern.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1 +1,32 @@
...
+import low from 'lowdb';
+import  { adapter } from './utils/dbutils';
+//TODO: find out why I can't import dotenv
+/* Load dotenv before code to apply environment variables */
+require('dotenv').config()
+//TODO: find out why I can't import AsciiTable
+const AsciiTable = require('ascii-table');
+
+/* Secret, Personal Telegram Token given by botfather */
+const token = process.env.BOT_TOKEN;
```
Comment (rafaelcalpena) [inline src/index.ts:15]: Use type: string

File: src/models/registry.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export class Resgistry {
```
Comment (rafaelcalpena) [inline src/models/registry.ts:1]: Typo

File: src/index.ts [hunk 3]
```diff
@@ -1 +1,32 @@
...
+import fs from 'fs';
+import strings from '../strings';
+import { setDefaultValues, mountDBObject } from './utils/dbutils';
+import low from 'lowdb';
+import  { adapter } from './utils/dbutils';
+//TODO: find out why I can't import dotenv
+/* Load dotenv before code to apply environment variables */
+require('dotenv').config()
+//TODO: find out why I can't import AsciiTable
+const AsciiTable = require('ascii-table');
```
Comment (rafaelcalpena) [inline src/index.ts:12]: Can you check if this object has a type definition please?

File: src/index.ts [hunk 4]
```diff
@@ -1 +1,32 @@
...
+/* Secret, Personal Telegram Token given by botfather */
+const token = process.env.BOT_TOKEN;
+
+/* DB-Related */
+const db = low(adapter);
+setDefaultValues(db);
+mountDBObject(db);
+
+/** Used for /debug command */
+let lastDebug = {}
```
Comment (rafaelcalpena) [inline src/index.ts:23]: Add a "TODO: Add types" here

File: src/index.ts [hunk 5]
```diff
@@ -1 +1,32 @@
...
+/** Used by admin to send bulk messages */
+let admBulkMessages = {}
```
Comment (rafaelcalpena) [inline src/index.ts:25]: Add a "TODO: Add types" here

**Comment by rafaelcalpena** [comment]
Related to #8

File: src/index.ts (hunk 1 above)
Comment (moreiravictor) [reply src/index.ts:15]: I'll have to user string | undefined for now, but I'll think of another way to deal with this later.

File: src/index.ts (hunk 3 above)
Comment (moreiravictor) [reply src/index.ts:12]: I don't think so. I searched for it, but didn't find any.

File: src/models/registry.ts (hunk 2 above)
Comment (moreiravictor) [reply src/models/registry.ts:1]: I've looked into the stringify method and it seems it doesn't work properly with classes but there's a way to implement it, like this: 
http://codebuckets.com/2018/06/17/extending-typescript-to-serialise-map-objects-to-json/
What do you think?

**Pull request merged**